README.md

```diff
@@ -127,8 +103,38 @@ Check out `./cockroach help` to see all available commands.
 ## Deploying CockroachDB in production
 
 To run a CockroachDB cluster on various cloud platforms using [docker machine](http://docs.docker.com/machine/),
-see [cockroach-prod](https://github.com/cockroachdb/cockroach-prod)
+see [cockroach-prod](https://github.com/cockroachdb/cockroach-prod).
+
+We'll set up a three-node cluster below.
```

Comment:
maybe not machine specific, but I would avoid the word "production" for this, it implies too many things. A separate sub-section called "multi-node cluster" might be better.